lib/libutee/include/tee_api_defines.h

```diff
@@ -192,6 +192,7 @@
 #define TEE_ALG_ECDSA_P256                      0x70003041
 #define TEE_ALG_ECDSA_P384                      0x70004041
 #define TEE_ALG_ECDSA_P521                      0x70005041
+#define TEE_ALG_ED25519                         0x70006043
```

Comment:
Please add a comment at the end of the define that this is from the v1.3 or v1.3.1 spec.

Reply:
Same for the other new defines in this commit.

Reply:
done